OsmAnd.xcodeproj/project.pbxproj

```diff
@@ -18474,6 +18507,7 @@
 				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "Dev OsmAnd Profile";
 				SUPPORTS_MACCATALYST = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "Sources/OsmAnd Maps-Bridging-Header.h";
+				SWIFT_OBJC_INTEROP_MODE = objc;
```

Comment:
Do you need to enable SWIFT_OBJC_INTEROP_MODE for this task? It's better to separate such changes into a separate commit